modules/round/src/main/Env.scala

```diff
@@ -125,6 +125,12 @@ final class Env(
 
   lazy val proxyRepo: GameProxyRepo = wire[GameProxyRepo]
 
+  scheduler.scheduleAtFixedRate(1 minute, 1 minute) { () =>
```

Comment:
For now polled every minute to test, will be once 24h in prod, but with restarts I think it will slowly shift, is it an issue?